tfidf-benchmark/Visualize.ipynb

```diff
@@ -0,0 +1,1558 @@
+{
```

Comment:
<p>Love this graph. Could you add cuml+Dask+n_gpus to the legend</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/rapids-examples/pull/31/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>